src/app/drive/components/EmptySharedView/EmptySharedView.tsx

```diff
@@ -0,0 +1,64 @@
+import { UploadSimple, Users } from '@phosphor-icons/react';
+import { useTranslationContext } from '../../../i18n/provider/TranslationProvider';
+import Empty from '../../../shared/components/Empty/Empty';
+import folderEmptyImage from 'assets/icons/light/folder-open.svg';
+
+interface EmptySharedViewProps {
+  onUploadFileButtonClicked: () => void;
+  sharedNamePath: string[];
+  isCurrentUserViewer: () => boolean;
+}
+
+const EmptySharedView = ({ onUploadFileButtonClicked, sharedNamePath, isCurrentUserViewer }) => {
```

Comment:
As it is specific to the SharedView, add it to the folder that belongs to this view, in share/components.